packages/components/src/InputPhoneNumber/InputPhoneNumber.tsx

```diff
@@ -41,9 +41,21 @@ export function InputPhoneNumber({
             value: Boolean(required),
             message: `${placeholder || "This"} is required`,
           },
+          validate: getPhoneNumberValidation,
           ...validations,
         }}
       />
     </InputMask>
   );
 }
+
+function getPhoneNumberValidation(value: string): string | true {
+  // Remove space, parenthesis and hyphen
+  const cleanValue = value.replace(/[- )(]/g, "");
+
+  if (cleanValue.length < 10) {
+    return "Enter a phone number";
+  }
+
+  return true;
```

Comment:
also need to call `validations.validate(value)` somewhere here in case someone does add it on the props